Libraries/Components/Clipboard/Clipboard.android.js

```diff
@@ -10,4 +10,6 @@
  */
 'use strict';
 
-module.exports = require('NativeModules').Clipboard;
```

Comment:
There's no need to have separate Clipboard.android.js and Clipboard.ios.js files if they're both identical. I suggest deleting one and renaming the other to Clipboard.js